i18n/en.json

```diff
@@ -55,17 +56,17 @@
 	"notification-header-request-declined": "Your request has been declined for the following reason:",
 	"requestwiki": "Request a wiki",
 	"requestwiki-edit-success": "Changes to request submitted successfully.",
-	"requestwiki-error-invalidcomment": "Your wiki request contains an invalid character. Please try requesting again, and if the problem persists [[Help center|please contact us]]",
-	"requestwiki-error-patient": "You have already requested a wiki. Please be patient and one of our wiki creators should create it soon. If you need to request another wiki, please wait and you will be able to request later.",
-	"requestwiki-header": "Thank you for choosing Miraheze to host your wiki! To get started, please fill out this form so we can review your request.\nYour wiki request will be reviewed by a volunteer, and it can be tracked at the page that will be given when submitting your request.\n\nIf you want a custom domain for your wiki (i.e. not a standard .miraheze.org subdomain), please read <span class=\"plainlinks\">[$1 this]</span> first. You must first choose a subdomain, and your custom domain will be configured afterwards. \n\nWhen you request a wiki, you automatically agree to our [[Content Policy|Content]] and [[Dormancy Policy]].",
+	"requestwiki-error-invalidcomment": "Your wiki request contains an invalid character. Please try requesting again.",
+	"requestwiki-error-patient": "You have already requested a wiki. Please be patient and we should create it soon. If you need to request another wiki, please wait and you will be able to request later.",
+	"requestwiki-header": "Thank you for choosing us to host your wiki! To get started, please fill out this form so we can review your request.\nYour wiki request will be reviewed and we will let you know if we decline or approve this request.",
 	"requestwiki-label-bio": "Will this wiki's main subject(s) be about living people?",
 	"requestwiki-label-language": "Language:",
 	"requestwiki-label-private": "Private wiki (not readable by everyone; only readable by users given the member or sysop groups):",
 	"requestwiki-label-purpose": "The purpose of this wiki is...",
 	"requestwiki-label-sitename": "Wiki name:",
-	"requestwiki-label-siteurl": "Subdomain: (subdomain.miraheze.org)",
+	"requestwiki-label-siteurl": "Subdomain:",
```

Comment:
This one can stay, I will make the domain a $1 variable based on configuration later.

Reply:
Done.